helm/fdb-operator/Chart.yaml

```
@@ -18,4 +18,4 @@ version: 0.1.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 1.16.0
+appVersion: 0.17.0
```

```suggestion
appVersion: 0.18.0
```

We bumped the version right around the time you posted this. 😀